src/client/components/app/Transactions/DeployLineTx.tsx

```diff
@@ -162,12 +190,17 @@ export const DeployLineTx: FC<DeployLineProps> = (props) => {
         })
       ).then((res) => {
         if (res.meta.requestStatus === 'rejected') {
-          // setTransactionCompleted(2);
           setLoading(false);
         }
         if (res.meta.requestStatus === 'fulfilled') {
+          const { lineAddress } = res.payload as {
+            lineAddress: string;
+            deployData: DeploySecuredLineWithConfigProps;
+          };
+          setLineAddress(lineAddress);
           setTransactionCompleted(1);
           setLoading(false);
+          history.push(`/${currentNetwork}/lines/${lineAddress}`);
```

Comment:
want all navigation to be done via `Link` component bc we have a lot of tracking logic in there. Failing that we also need to def use `logAppAnalytics` action?

Reply:
fixed to use `Link` component and established a pattern to eventually remove all uses of `history.push`